### PR TITLE
Initial application setup

### PR DESCRIPTION
**Pull request by evgenybutkevich**

### CODE REVIEW

File: app/controllers/players/create.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+        player.validate()
+            .then(({ validations }) => {
+                if (validations.get('isValid')) {
+                    if (!confirm('Are you sure?!')) {
+                        return;
+                    }
+                }
+            });
+
+        player.save()
```
Comment (Egorrr) [inline app/controllers/players/create.js]: Save is performed independently of validation result. please fix that

File: app/controllers/players/create.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+            .then(() => {
+                this.transitionToRoute('players.index');
+            }, () => { }
+            );
```
Comment (Egorrr) [inline app/controllers/players/create.js]: What is this empty function is for?

File: app/routes/users/create.js [hunk 3]
```diff
@@ -0,0 +1,7 @@
+import Route from '@ember/routing/route';
+
+export default class CreateRoute extends Route {
+    model() {
+        return this.store.createRecord('user', { });
```
Comment (Egorrr) [inline app/routes/users/create.js]: No need to pass empty object to createRecord as far as I remember

File: app/styles/components/nav-bar.css [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+	background-color: lightgray;
+	left: 480px;
+	opacity: 0.4;
+	width: 120px;
+}
+
+nav a:nth-child(6):hover~.animation {
+	background-color: lightgray;
+	left: 600px;
+	opacity: 0.4;
```
Comment (Egorrr) [inline app/styles/components/nav-bar.css:103]: Please use .scss styles instead, check their possibilities and use them to reduce duplicated styles here

File: app/styles/content/button.css [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+    border: none;
+
+    color: white;
+    font-size: 20px;
+    font-weight: bold;
+    margin: 2px 50px;
+    opacity: 0.9;
+
+    text-decoration: none;
+    width: 100px; */
```
Comment (Egorrr) [inline app/styles/content/button.css]: Please don't leave commented styles as well as commented code

File: app/styles/content/content.css [hunk 6]
```diff
@@ -0,0 +1,8 @@
+.content-background {
+    background-color: lightgray;
+}
+
+.content-div {
+    margin: 0 auto;
+    max-width: 90%;
+}
```
Comment (Egorrr) [inline app/styles/content/content.css:8]: What is the content/content?

File: app/validators/username-available.js [hunk 7]
```diff
@@ -0,0 +1,39 @@
+import BaseValidator from 'ember-cp-validations/validators/base';
```
Comment (Egorrr) [inline app/validators/username-available.js]: Do you use this custom validator?

File: app/adapters/application.js [hunk 8]
```diff
@@ -0,0 +1,5 @@
+import RESTAdapter from '@ember-data/adapter/rest';
+
+export default class ApplicationAdapter extends RESTAdapter {
+    host = 'http://localhost:3000';
```
Comment (AlexanderShcherbakov) [inline app/adapters/application.js]: please move the `host` to env config

File: app/components/players/input.hbs [hunk 9]
```diff
@@ -0,0 +1,22 @@
...
+    <label for="lastName">Last name:</label>
+    {{input id="lastName" placeholder="Last name" value=@player.lastName}}
+</div>
+<div class='error'>
+    {{v-get @player 'lastName' 'message'}}
+</div>
+
+{{#if @player.id}}
+    <button {{on "click" @onUpdate}}>Save</button>
+    {{!-- <button {{on "click" (fn @onDelete @player.id)}}>Delete</button> --}}
```
Comment (AlexanderShcherbakov) [inline app/components/players/input.hbs]: please remove commented code

File: app/styles/components/nav-bar.css (hunk 4 above)
Comment (evgenybutkevich) [reply app/styles/components/nav-bar.css:103]: I will gradually replace .css style files

File: app/styles/content/content.css (hunk 6 above)
Comment (evgenybutkevich) [reply app/styles/content/content.css:8]: I'll rework it with scss

**Review by Egorrr — APPROVED** [review]

**Pull request merged**